segments/git_stash.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+
+
+def add_git_stash_segment():
+    stash_count = subprocess.check_output('git stash list | wc -l', shell=True).strip()
+
+    if stash_count == '0':
+        return
+
+    bg = Color.REPO_DIRTY_BG
+    fg = Color.REPO_DIRTY_FG
```

Comment:
A pull request that I merged this evening began adding different colors specifically for the different things git might show. I think you can probably do the same here rather than repurpose these colors.